<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActiveTrainning</class>
 <widget class="QWidget" name="ActiveTrainning">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1692</width>
    <height>944</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="startTrain_Btn">
   <property name="geometry">
    <rect>
     <x>1502</x>
     <y>833</y>
     <width>140</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 140px;
height: 70px;
background: #0D9DDB;
border-radius: 8px;

width: 64px;
height: 32px;
font-size: 32px;
font-family: SourceHanSansCN-Medium, SourceHanSansCN;
font-weight: 500;
color: #FFFFFF;
line-height: 48px;
</string>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QLabel" name="trainSelect_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>37</y>
     <width>240</width>
     <height>106</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 104px;
height: 26px;
font-size: 26px;
font-family: SourceHanSansCN-Regular, SourceHanSansCN;
font-weight: 400;
color: #333333;
line-height: 39px;
background: #E4F4FC;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>       训练选择</string>
   </property>
  </widget>
  <widget class="QLabel" name="gameForm_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>145</y>
     <width>240</width>
     <height>106</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 104px;
height: 26px;
font-size: 26px;
font-family: SourceHanSansCN-Regular, SourceHanSansCN;
font-weight: 400;
color: #333333;
line-height: 39px;
background: #E4F4FC;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>       游戏形式</string>
   </property>
  </widget>
  <widget class="QLabel" name="trainScope_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>253</y>
     <width>240</width>
     <height>106</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 104px;
height: 26px;
font-size: 26px;
font-family: SourceHanSansCN-Regular, SourceHanSansCN;
font-weight: 400;
color: #333333;
line-height: 39px;
background: #E4F4FC;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>       训练范围</string>
   </property>
  </widget>
  <widget class="QLabel" name="trainTime_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>361</y>
     <width>240</width>
     <height>106</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 104px;
height: 26px;
font-size: 26px;
font-family: SourceHanSansCN-Regular, SourceHanSansCN;
font-weight: 400;
color: #333333;
line-height: 39px;
background: #E4F4FC;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>       训练时间</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="varusValgus_RadioButton">
   <property name="geometry">
    <rect>
     <x>552</x>
     <y>65</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 58px;
height: 29px;
font-size: 29px;
font-family: SourceHanSansCN-Regular, SourceHanSansCN;
font-weight: 400;
color: #000000;
line-height: 44px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="plantarFlexion_RadioButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>64</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="comprehensive_RadioButton">
   <property name="geometry">
    <rect>
     <x>752</x>
     <y>64</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 58px;
height: 29px;
font-size: 29px;
font-family: SourceHanSansCN-Regular, SourceHanSansCN;
font-weight: 400;
color: #000000;
line-height: 44px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>67</y>
     <width>46</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/images/activitypath.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>175</y>
     <width>46</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/gameform.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>283</y>
     <width>46</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/activityscope.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>389</y>
     <width>46</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/traintime.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>142</y>
     <width>1220</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 1220px;
height: 1px;
background: #C3E4F3;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>1220</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 1220px;
height: 1px;
background: #C3E4F3;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>358</y>
     <width>1220</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 1220px;
height: 1px;
background: #C3E4F3;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>466</y>
     <width>1220</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 1220px;
height: 1px;
background: #C3E4F3;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>377</x>
     <y>384</y>
     <width>260</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #F4F4F4;
border-radius: 30px;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="timeSubtract_btn">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>4</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/minus.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="timeAdd_btn">
    <property name="geometry">
     <rect>
      <x>198</x>
      <y>6</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文细黑</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/plus.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="time_lineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SourceHanSansCN-Regular</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 30px;
font-family: SourceHanSansCN-Regular, SourceHanSansCN;
font-weight: 400;
color: #333333;
line-height: 45px;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="cursorPosition">
     <number>1</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>657</x>
     <y>392</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>min</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="slopePlantar_RadioButton">
   <property name="geometry">
    <rect>
     <x>952</x>
     <y>64</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 58px;
height: 29px;
font-size: 29px;
font-family: SourceHanSansCN-Regular, SourceHanSansCN;
font-weight: 400;
color: #000000;
line-height: 44px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>49</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/images/trainselect/zhiqubeiqu.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>592</x>
     <y>50</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/images/trainselect/neifanwaifan.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>792</x>
     <y>50</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/images/trainselect/yuanzhou.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>992</x>
     <y>50</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/images/trainselect/xiezhiqubeiqu.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="gameType_ListWidget">
   <property name="geometry">
    <rect>
     <x>358</x>
     <y>146</y>
     <width>1061</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>114</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>趾屈背屈</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>562</x>
     <y>114</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>内翻外翻</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>766</x>
     <y>114</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>圆周运动</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>114</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>斜趾屈背屈</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="adductionAbduction_RadioButton">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>62</y>
     <width>21</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 58px;
height: 29px;
font-size: 29px;
font-family: SourceHanSansCN-Regular, SourceHanSansCN;
font-weight: 400;
color: #000000;
line-height: 44px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>47</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/images/trainselect/neishouwaizhan.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>110</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>内收外展</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="trainScope_GroupBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>971</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="maxValue_GroupBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>451</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>182</x>
       <y>10</y>
       <width>260</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #F4F4F4;
border-radius: 30px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="rightAngleSubtract_btn">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>5</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/minus.png);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="rightMaxAngle_lineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>0</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>SourceHanSansCN-Regular</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 30px;
font-family: SourceHanSansCN-Regular, SourceHanSansCN;
font-weight: 400;
color: #333333;
line-height: 45px;</string>
      </property>
      <property name="text">
       <string>20°</string>
      </property>
      <property name="cursorPosition">
       <number>3</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="rightAngleAdd_btn">
      <property name="geometry">
       <rect>
        <x>185</x>
        <y>5</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文细黑</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/plus.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>132</x>
       <y>34</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>一一</string>
     </property>
    </widget>
    <widget class="QLabel" name="minAngle_label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>25</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 17px;
height: 30px;
font-size: 30px;
font-family: SourceHanSansCN-Regular, SourceHanSansCN;
font-weight: 400;
color: #333333;
line-height: 45px;</string>
     </property>
     <property name="text">
      <string>0°</string>
     </property>
    </widget>
    <widget class="QLabel" name="right_Label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>80</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>背屈</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="minValue_GroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>451</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>182</x>
       <y>10</y>
       <width>260</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #F4F4F4;
border-radius: 30px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="leftAngleSubtract_btn">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>5</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/minus.png);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="leftMaxAngle_lineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>0</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>SourceHanSansCN-Regular</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 30px;
font-family: SourceHanSansCN-Regular, SourceHanSansCN;
font-weight: 400;
color: #333333;
line-height: 45px;</string>
      </property>
      <property name="text">
       <string>20°</string>
      </property>
      <property name="cursorPosition">
       <number>3</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="leftAngleAdd_btn">
      <property name="geometry">
       <rect>
        <x>184</x>
        <y>5</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华文细黑</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/plus.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>132</x>
       <y>34</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>一一</string>
     </property>
    </widget>
    <widget class="QLabel" name="minAngle_label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>25</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 17px;
height: 30px;
font-size: 30px;
font-family: SourceHanSansCN-Regular, SourceHanSansCN;
font-weight: 400;
color: #333333;
line-height: 45px;</string>
     </property>
     <property name="text">
      <string>0°</string>
     </property>
    </widget>
    <widget class="QLabel" name="left_Label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>80</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>趾屈</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>530</y>
     <width>1331</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> border:none;</string>
   </property>
   <property name="title">
    <string>提示!</string>
   </property>
   <widget class="QLabel" name="tips_Label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>1281</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 26pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <zorder>trainScope_GroupBox</zorder>
  <zorder>startTrain_Btn</zorder>
  <zorder>trainSelect_label</zorder>
  <zorder>gameForm_label</zorder>
  <zorder>trainScope_label</zorder>
  <zorder>trainTime_label</zorder>
  <zorder>label</zorder>
  <zorder>label_4</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>line_3</zorder>
  <zorder>line_4</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>label_8</zorder>
  <zorder>label_2</zorder>
  <zorder>label_9</zorder>
  <zorder>label_13</zorder>
  <zorder>gameType_ListWidget</zorder>
  <zorder>label_3</zorder>
  <zorder>label_14</zorder>
  <zorder>label_16</zorder>
  <zorder>label_17</zorder>
  <zorder>adductionAbduction_RadioButton</zorder>
  <zorder>label_11</zorder>
  <zorder>label_15</zorder>
  <zorder>varusValgus_RadioButton</zorder>
  <zorder>plantarFlexion_RadioButton</zorder>
  <zorder>label_12</zorder>
  <zorder>comprehensive_RadioButton</zorder>
  <zorder>slopePlantar_RadioButton</zorder>
  <zorder>groupBox_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
